config/locales/en.yml

```diff
@@ -872,6 +873,83 @@ en:
         police_ongoing_proceedings: Evidence of relevant criminal proceedings for a domestic violence offence which have not concluded
         police_conviction: Evidence of a relevant conviction for a domestic violence offence
         police_dvpn: A domestic violence protection notice issued under section 24 of the Crime and Security Act 2010 against a prospective party
+        ## GROUP COURT ##
```

Comment:
Just a suggestion: break the locales out categorically:

```
locales/en/group_police.yml
locales/en/police.yml
```
I find them easier to maintain this way, personally.

Reply:
Yeah I know. In tax tribs we originally had multiple i18n files, but then we decided to unify all of them into a big monolith because for shared keys it simplified the work.

But again, tax tribs was a baby compared to this project, so yes, I'm every other day trying to decide if we should split the i18n or not and how to best organise it (I'm not a big fan of many small i18n but a few is ok, for example one for exemptions seems quite logical).

Reply:
Maybe it's worth looking at one of the Rails `gettext` implementations at this point?